### PR TITLE
Add prepare() function to initialize camera plugin

### PR DESCRIPTION
**Pull request by JonasVautherin**
Depends on https://github.com/mavlink/MAVSDK-Proto/pull/255

The new `prepare()` function tries to fetch the camera definition and returns `Success` or `Error`. In case of `Error`, calling `prepare()` again will try again.

### CODE REVIEW

File: src/plugins/camera/mocks/camera_mock.h [hunk 1]
```diff
@@ -7,6 +7,7 @@ namespace testing {
 
 class MockCamera {
 public:
+    MOCK_CONST_METHOD0(prepare, Camera::Result()){};
```
Comment (julianoes) [inline src/plugins/camera/mocks/camera_mock.h:10]: But there is currently no result?

File: src/plugins/camera/camera_impl.cpp [hunk 2]
```diff
@@ -120,6 +120,14 @@ void CameraImpl::deinit()
     _camera_found = false;
 }
 
+void CameraImpl::prepare()
+{
+    // This is a trick to get mavsdk_server to initialize the camera plugin. Because mavsdk_server
+    // only initializes plugins when they are first used (LazyPlugin), and this may end up timing
+    // out on the first call of a camera function. By calling this before, one can ensure that the
+    // camera plugin is initialized early.
```
Comment (julianoes) [inline src/plugins/camera/camera_impl.cpp:128]: Oh funny, you don't actually do anything in here. I thought you would kick off something.

Comment (JonasVautherin) [reply src/plugins/camera/camera_impl.cpp]: Yeah I thought it was clever, but you're right. I changed it to actually return `Success` when the camera_definition has been downloaded.

Comment (JonasVautherin) [reply src/plugins/camera/camera_impl.cpp]: And it will retry downloading the camera_definition every time you call `prepare()`.

**Review by julianoes — DISMISSED** [review]

**Pull request merged**